tools/submission/submission-checker.py

```diff
@@ -0,0 +1,554 @@
+"""
+A checker for mlperf inference submissions
+"""
+
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import argparse
+import json
+import logging
+import os
+import re
+import sys
+
+# pylint: disable=missing-docstring
+
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger("main")
+
+
+MODEL_CONFIG = {
+    "v0.5": {
+        "models": ["ssd-small", "ssd-large", "mobilenet", "resnet", "gnmt"],
+        "required-scenarios-datacenter": {
+            # anything goes
+        },
+        "required-scenarios-edge": {
+            # anything goes
+        },
+        "accuracy-target": {
+            "mobilenet": ("acc", 71.68 * 0.98),
+            "resnet": ("acc", 76.46 * 0.99),
+            "ssd-small": ("mAP", 22 * 0.99),
+            "ssd-large": ("mAP", 20 * 0.99),
+            "gnmt": ("bleu", 23.9 * 0.99),
+        },
+        "performance-sample-count": {
+            "mobilenet": 1024,
+            "resnet": 1024,
+            "ssd-small": 256,
+            "ssd-large": 64,
+            "gnmt": 3903900,
+        },
+        "seeds": {
+            "qsl_rng_seed": 3133965575612453542,
+            "sample_index_rng_seed": 665484352860916858,
+            "schedule_rng_seed": 3622009729038561421,
+        },
+    },
+    "v0.7": {
+        "models": ["ssd-large", "resnet", "rnnt", "3d-unet", "dlrm", "bert"],
+        "required-scenarios-datacenter": {
+            "resnet": ["Server", "Offline"],
+            "ssd-large": ["Server", "Offline"],
+            "rnnt": ["Server", "Offline"],
+            "bert": ["Server", "Offline"],
+            "dlrm": ["Server", "Offline"],
+            "3d-unet": ["Offline"],
+        },
+        "required-scenarios-edge": {
+            "resnet": ["Server", "Offline"],
+            "ssd-large": ["Server", "Offline"],
+            "rnnt": ["Server", "Offline"],
+            "bert": ["Server", "Offline"],
+            "dlrm": ["Server", "Offline"],
+            "3d-unet": ["Offline"],
+        },
+        "accuracy-target": {
+            "resnet": ("acc", 76.46 * 0.99),
+            "ssd-large": ("mAP", 20 * 0.99),
+            "rnnt": ("WER", 7.452 * 0.99),
+            "bert": ("F1", [90.874 * 0.99, 90.874 * 0.999]),
+            "dlrm": ("AUC", [76.46 * 0.99, 76.46 * 0.999]),
+            "3d-unet": ("mean", [0.853 * 0.99, 0.853 * 0.999]),
+        },
+        "performance-sample-count": {
+            "ssd-large": 64,
```

Comment:
we need ssd-mobilenet as well

Reply:
v0.5 has a separate section  and it has ssd-small which ssd-mobilenet will be mapped to.
For v0.7 I belief ssd-mobilenet [is out](https://github.com/mlperf/inference_policies/blob/master/inference_rules.adoc#41-benchmarks)

Reply:
Still in for the Edge.

Reply:
My bad ... adding it.